guides/v2.1/cloud/project/project-conf-files_routes.md

```diff
@@ -54,17 +54,18 @@ For example, if the project ID is `mswy7hzcuhcjw` on a branch called `refactorcs
 
 <!-- {{site.data.var.ece}} also supports [multiple applications]({{ page.baseurl }}/cloud/project/project-conf-multi.html) per project. Each project has a single `routes.yaml` file that defines which request is routed to which application. -->
 
-## Route options {#cloud-yaml-routes-opts}
+## Route options
 
 Configure each route separately using the following properties:
 
--  `type: upstream`—serves an application. Also, it has an `upstream` property that specifies the name of the application (as defined in `.magento.app.yaml`) followed by the `:http` endpoint.
--  `type: redirect`—redirects to another route. It is followed by the `to` property, which is an HTTP redirection to another route identified by its template.
--  `cache`—controls [caching for the route]({{ page.baseurl }}/cloud/project/project-routes-more-cache.html).
--  `redirects`—controls [redirect rules]({{ page.baseurl }}/cloud/project/project-routes-more-redir.html).
--  `ssi`—controls enabling of [Server Side Includes]({{ page.baseurl }}/cloud/project/project-routes-more-ssi.html).
+`type: upstream` | Serves an application. Also, it has an `upstream` property that specifies the name of the application (as defined in `.magento.app.yaml`) followed by the `:http` endpoint.
```

Comment:
Using pipes to separate property from description in a list looks like a new style trend. I'm not opposed, but don't we normally use em dashes?

Reply:
Oops! I didn't even think about it being a table. If it renders as a table, then please disregard.